shared/domains/chelonia/chelonia.js

```diff
@@ -177,7 +177,7 @@ const rawSignatureFnBuilder = (key) => {
   }
 }
 
-const signatureFnBuilder = function (this: any, signingContractID, signingKeyId) {
+const signatureFnBuilder = function (signingContractID: string, signingKeyId: string) {
```

Comment:
This file must not be modified in this PR, so when re-creating the PR please make sure no files under `shared/domains/chelonia` are modified